pa11ycrawler/cmdline.py

```diff
@@ -198,6 +204,10 @@ def main():
     parser = cli()
     args = parser.parse_args()
     settings = args_to_settings(args)
+    logging.basicConfig(
+        format='%(asctime)s [%(levelname)s] [%(name)s]: %(message)s',
+        level=settings.get('LOG_LEVEL')
```

Comment:
The default is set to INFO in [settings.py](https://github.com/edx/pa11ycrawler/blob/master/pa11ycrawler/settings.py#L91).